### ISSUE TITLE
Add Discord notifications

### ISSUE THREAD

**Issue by axiomatic-aardvark** [graphops/graphcast-sdk#50]
We need an option for Discord notifications, similar to the Slack implementation we currently have. The simplest way of doing this would be [webhooks](https://discord.com/developers/docs/resources/webhook).

**Comment by alex-pakalniskis** [comment]
Hey @axiomatic-aardvark,

Hope this helps as a starting point for `graphops/graphcast-sdk/src/discord_bot.rs` development
``` rs
use std::error::Error;
use std::collections::HashMap;


#[derive(Debug, Clone, PartialEq, Eq)]
pub struct DiscordBot {}

impl DiscordBot {
    pub async fn send_webhook(
        channel: &String,
        radio_name: String,
        content: String,
    ) -> Result<(), Box<dyn Error>> {

        let map = HashMap::new();
        map.insert("content", format!("Hi there! on the topic of {}: \n{}", radio_name, content));

        let client = reqwest::blocking::Client::new();

        let res = client.post(channel)
            .json(&map)
            .send()?;
    }
}
```

Cheers :smile:

**Comment by hopeyen** [comment]
Resolved in #136 by  @axiomatic-aardvark 🍻
